src/EA.Weee.Web/Areas/AatfReturn/Controllers/SelectYourPCSController.cs

```diff
@@ -37,7 +37,7 @@ public SelectYourPcsController(Func<IWeeeClient> apiclient, BreadcrumbService br
         }
 
         [HttpGet]
-        public virtual async Task<ActionResult> Index(Guid organisationId, Guid returnId, bool reselect = false)
+        public virtual async Task<ActionResult> Index(Guid organisationId, Guid returnId, bool reselect = false, bool copyPrevious = false, bool clearSelections = false)
         {
             if (reselect)
             {
```

Comment:
Think we need to refactor this a little can we move the code from the Reselect method into the main Index action as we will need to do the copy functionality if its a reselection, currently there is an exception if you come from the Change your pcs selection link on the recieved pcs list

So set the SelectedSchemes  and reselect properties in here.